File: backoffice/src/main/resources/static/js/common/utils/StringUtils.js
/**
 *
 */
class StringUtils {

	constructor ( ) {

	}

	static isEmpty ( value ) {
		let _result = true;
		if ( value != null && typeof value != 'undefined' ) {
			const str = (value).toString().trim();
			if ( str.length > 0 ) {
				_result = false;
			}
		}
		return _result;
	}

	static isNotEmpty ( value ) {
		return !this.isEmpty(value);
	}

	static nvl ( value, replaceValue ) {
		return this.isEmpty(value) ? replaceValue : value;
	}

}

File: backoffice/src/main/java/co/kr/store/bo/common/utils/StringUtils.java
package co.kr.store.bo.common.utils;

public class StringUtils {

	public static boolean isEmpty(String str) {
		return org.apache.commons.lang3.StringUtils.isEmpty(org.apache.commons.lang3.StringUtils.trim(str));
	}

	public static boolean isNotEmpty(String str) {
		return !StringUtils.isEmpty(str);
	}

	public static boolean equals(String str, String compareStr) {
		return org.apache.commons.lang3.StringUtils.equals(str, compareStr);
	}

	public static String nvl(String str) {
		return StringUtils.nvl(str, org.apache.commons.lang3.StringUtils.EMPTY);
	}

	public static String nvl(String str, String compareStr) {
		if ( StringUtils.isEmpty(str) ) {
			return compareStr;
		}
		return str;
	}

}
